src/cdk-experimental/menu/menu.md

```diff
@@ -0,0 +1,240 @@
+
+#### Menu panels
+
+Non-inline menus must be wrapped in an `ng-template` which must have a directive of `cdkMenuPanel`
+and a reference variable which must be of type `cdkMenuPanel`. Further, the `cdkMenu` must also
+reference the `cdkMenuPanel`.
+
+```html
+<ng-template cdkMenuPanel #panel="cdkMenuPanel">
+  <div cdkMenu [cdkMenuPanel]="panel">
+    <!-- some content -->
+  </div>
+</ng-template>
+```
+
+Finally, note that this is part of the cdk and there is no styling - it is up to you to provide
+styles. It is also important that your style matches the `orientation` attribute on both the
+`cdkMenu` and `cdkMenuBar` directives.
+
+### Standalone Menus
+
+A standalone menu is triggered from a single standalone button and follows the [ARIA menu][menubar]
+spec. Standalone menu triggers cannot be grouped together meaning each single trigger is a tabstop.
+Standalone triggers can be opened from the keyboard and the opened menu behaves just like it does in
+the other implementations.
+
+<!-- TODO standalone trigger example (use from above?) -->
+
+### Menu Bars
+
+The `CdkMenuBar` directive follows the [ARIA menubar][menubar] spec and operates similar to a
+desktop app menubar. It consists of at least one `CdkMenuItem` which triggers a submenu. A menubar
+can be layed out horizontally or vertically (defaulting to horizontal). If the style is changed, the
+`orientation` attribute must be set to match in order for the keyboard navigation to work properly
+and for menus to open up in the correct location.
+
+The primary difference between a menubar and a standalone menu is the menubar groups together the
+triggering components. Therefore, menus in a menubar can be opened/closed when hovering between
+their triggers and keyboard navigation listens to left and right arrow buttons which toggles between
+the menu items in the menubar.
+
+<!-- TODO basic menubar example (google docs?) -->
+
+### Context Menus
+
+A context menu is a menu which opens on a right click which occurs within some context component. A
+context is an area of your application which contains some content where you want to open up a menu.
+When a user right-clicks within the context the associated menu opens up next to their cursor - as
+you would expect with a typical right click menu.
+
+<!-- TODO basic context menu example -->
+
+Context menu triggers may be nested within parent contexts and when a user right clicks, the deepest
+context menu in which the user is hovering in is opened up.
+
+```html
+<div [cdkContextMenuTriggerFor]="outer">
+  My outer context
+  <div [cdkContextMenuTriggerFor]="inner">
+    My inner context
+  </div>
+</div>
+```
+
+In the example above, right clicking on "My inner context" will open up the "inner" menu and right
+clicking inside "My outer context" will open up the "outer" menu.
+
+### Inline Menus
+
+Inline menus are groups of menu items which typically do not trigger their own menus and the menu
```

Comment:
This could also be clearer -- emphasize that the menu doesn't 'pop up', but all menu items are displayed from the start.